app-crypt/gnupg/gnupg-2.2.27.recipe

```diff
@@ -6,20 +6,16 @@ well as access modules for all kinds of public key directories. GnuPG, also \
 known as GPG, is a command line tool with features for easy integration with \
 other applications."
 HOMEPAGE="https://gnupg.org/"
-COPYRIGHT="1998-2018 Free Software Foundation, Inc.
-	2000, Dimitrios Souflis
-	2008, 2009, 2010, 2012-2016  William Ahern"
-LICENSE="CC0 v1.0
-	GNU GPL v2
-	GNU GPL v3
-	GNU LGPL v2.1
+COPYRIGHT="1997-2019 Werner Koch
+	1994-2021 Free Software Foundation, Inc."
+LICENSE="GNU GPL v3
```

Comment:
For Copyright, I think we should list the copyright holders also in here? (see AUTHORS file in the source)
For License, I think we should keep the list as it was (also found in the AUTHORS file)